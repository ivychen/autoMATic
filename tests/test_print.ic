/* autoMATic Compiler - Spring 2018 - COMS4115 PLT
 by Ivy Chen ic2389, Nguyen Chi Dung ncd2118,
 Nelson Gomez ng2573, Jimmy O'Donnell jo2474 */

int main() {
    int x = 5;
    float f = 4.2;
    string s = "whomp";
    int r;
    int matrix M;
    M = [[1,2];[3,2];[3,3]];
    println(x);
    println(f);
    println(s);
    println(rows(M)); // 3
    println(cols(M)); // 2
    r = rows(M);
    println(r); // 3
    return 0;
}
