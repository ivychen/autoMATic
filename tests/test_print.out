5
4.200000
whomp
3
2
3
